.gitlab-ci-build-openmr.yml

```diff
@@ -26,7 +26,7 @@ build:
     - echo "Building the image"
     - export TAG_NAME=$CI_COMMIT_REF_SLUG-`git show -s --format=%ct HEAD`-`git rev-parse --short=8 HEAD`
     - echo $TAG_NAME
-    - docker build -t $ECR_URI_BASE/offerland/$SERVICE_NAME:$TAG_NAME -t $ECR_URI_BASE/offerland/$SERVICE_NAME:$CI_COMMIT_REF_SLUG-latest .
+    - docker build -t $ECR_URI_BASE/offerland/$SERVICE_NAME:$TAG_NAME -t $ECR_URI_BASE/offerland/$SERVICE_NAME:$CI_COMMIT_REF_SLUG-latest --build-arg SENTRY_AUTH_TOKEN=${SENTRY_AUTH_TOKEN:-} --build-arg SENTRY_ENABLED=${SENTRY_ENABLED:-} .
```

Comment:
This param is not set across all repos; just make sure the docker `--build-args` flag works fine with null/empty values.

Reply:
@amir-abdi jan, thanks for your review.
I have checked it locally and on the pipeline and it works fine.